scripts/main.star

```diff
@@ -33,7 +33,8 @@ load(
     'publish_packages_step',
     'notify_pipeline',
     'integration_test_services',
-    'upload_cdn'
+    'upload_cdn',
+    'validate_scuemata'
```

Comment:
Are you forgetting to include the step in the pipeline?

Reply:
Yes